Makefile

```diff
@@ -382,7 +384,7 @@ endif
 WARNINGS = all error strict-prototypes missing-prototypes nested-externs \
 	missing-declarations undef pointer-arith no-nonnull declaration-after-statement
 
-CFLAGS += --std=c99 -nostdlib -nostdinc -ffreestanding \
+CFLAGS += --std=c99 -nostdlib -nostdinc -ffreestanding -fms-extensions \
```

Comment:
What is your plan going forward? The kernel sticks to relatively small subset of C, but the user libraries (including libsel4) assume very modern versions of gcc. Every architecture seL4 supports has an easy to acquire version of gcc that will build it. Instead of embarking on this journey of building with an old gcc on a case insensitive file system why do you not just build on another system (in a virtual machine for instance) where this will just work?